openai_assistant/test/test_annotations.py

```diff
@@ -88,3 +88,8 @@ def test_has_annotations(self):
 
         result = OpenAIAdapter(message_with_citation).has_annotations()
         self.assertTrue(result)
+
+    async def test_cache(self):
```

Comment:
Tests that aren't mocked should be named `test_cache_integration`. otherwise mock the API client call. See test_on_chat_resume_logic for a way to do this.